feat: allow handlers to provide arbitrary format for the error responses (#15)
deliveroo/jsonrest-go: README.md

If an error is returned, it will be sanitized and returned to the client as
json. Errors generated by a call to `jsonrest.Error(status, code, message)`
will be rendered in the following form:
```
{
    "error": {
        "message": message,
        "code": code
    }
}
```
along with the given HTTP status code.

> If the error returned supports `StatusCode() int` method, it will be marshaled as-is to the client, using standart `json.Marshal()` with the status code provided as well.

Reviewer: tipo: `standart -> standard`

> along with the given HTTP status code.
> If the error returned supports `StatusCode() int` method, it will be marshaled as-is to the client, using standard `json.Marshal()` with the status code provided as well.

Reviewer 2: nitpick: consistent line-wrapping.
```suggestion
along with the given HTTP status code.

If the error returned implements HTTPErrorResponse (i.e. has a `StatusCode()
int` method), it will be marshaled as-is to the client with the provided status
code.
```
